lab_2/main.py

```diff
@@ -0,0 +1,232 @@
+from operator import add
+import numpy as np
+import re
+import scipy.stats as stat
+from rpy2.robjects.packages import importr
+import rpy2.robjects as ro
+
+
+def F_PP(Input):
+    F = []
+    P = []
+    rows = len(Input)
+    cols = len(Input[0])
+    for i in range(rows):
+        pSearch = re.search('-P', Input[i][0])
+        fSearch = re.search('-F', Input[i][0])
+        if (fSearch):
+            F.append(Input[i])
+        if (pSearch):
+            P.append(Input[i])
+    return F, P
+
+
+def FP_Pairing(F, P):
+    Frow = len(F)
+    Prow = len(P)
+    cols = len(F[0])
+    pairs = list()
+    noMatch = [u'Unmatched with corresponding PP']
+    if (len(F[0]) != len(P[0])):
+        print
+        'Error, inputs have different numnber of columns'
```

Comment:
использовать современный синтаксис:
```suggestion
      print('Error, inputs have different number of columns')
```